force-app/main/default/classes/LookUpController.cls

```diff
@@ -0,0 +1,29 @@
+public with sharing class LookUpController {
+
+    @AuraEnabled(Cacheable = true)
+    public static List<SObject> getRecordList(String addInfo, String sObjectName, String searchKey){
+        String key = '%' + searchKey + '%';
```

Comment:
@TsSlava please use string methods and make it more secure
https://trailhead.salesforce.com/en/content/learn/modules/secure-serverside-development/mitigate-soql-injection